src/lib/Form/DataTransformer/LocationsTransformer.php

```diff
@@ -27,19 +30,48 @@ public function __construct(LocationService $locationService)
         $this->locationService = $locationService;
     }
 
+    /**
+     * Transforms a domain specific Location objects into a Location's ID comma separated string.
+     *
+     * @param mixed $value
+     *
+     * @return array|string
```

Comment:
Why is this returning an `array` or a `string`? I'd expect this to return a string (either comma separated ids or an empty one) or `null`.